public/app/features/plugins/admin/helpers.ts

```diff
@@ -103,6 +103,7 @@ export function mapLocalToCatalog(plugin: LocalPlugin, error?: PluginError): Cat
     hasUpdate,
   } = plugin;
 
+  const isDisabled = !!error || isDisabledSecretsPlugin(type);
```

Comment:
**Question:** just to see if I understand correctly - it means that there are going to be no errors coming from this plugin in case it is disabled, right? (And that is why we need this extra check I guess)

Reply:
Correct. Plugin can be installed, but disabled because it hasn't been configured or there's an invalid license. In the future (exactly when is tbd), I understand that we will be looking at ways to make some of these "service/provider plugin types" more generic so we don't have to have custom logic for each one.